rust/foxglove/src/websocket/ws_protocol/parameter.rs

```diff
@@ -34,7 +34,12 @@ pub enum ParameterType {
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 #[serde(untagged)]
 pub enum ParameterValue {
+    /// An integer value.
+    /// NOTE: This needs to be before `Number` to ensure that values without a fractional part
+    /// are preferrentially deserialized as integers, not floats.
```

Comment:
Note that the `///` docstrings on public types are compiled into [the user-facing documentation](https://docs.rs/foxglove/latest/foxglove/websocket/enum.ParameterValue.html). For internal commentary, we should use `//`:

```suggestion
    // NOTE: This needs to be before `Number` to ensure that values without a fractional part
    // are preferrentially deserialized as integers, not floats.
```

(This suggestion is probably moot, though, considering we're going to need to roll our own `Deserialize` implementation.)